Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenu.cpp

```diff
@@ -80,6 +86,11 @@ namespace AzToolsFramework
                     m_menu->addSeparator();
                     break;
                 }
+            case MenuItemType::Widget:
+                {
+                    m_menu->addAction(elem.second.m_widgetAction);
+                    break;
+                }
```

Comment:
for some of these cases are the braces necessary.

Reply:
They are not necessary for compilation, but I tend to add them as an extension of the curly brackets policy in the style guide
https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#use-of-curly-braces
(It is not listed and the switch without brackets is considered acceptable, but I tend to use them anyways as a habit)